src/components/ListItem.js

```diff
@@ -76,10 +76,10 @@ const ListItem = () => {
   const searchItems = async (e) => {
     let docSnap = await getDoc(currentCollectionRef);
     const data = docSnap.data().items;
-    const filteredItems = [];
+    let filteredItems = [];
 
     for (const item of data) {
-      console.log(item);
+      if (!e.target.value.length) filteredItems = data;
```

Comment:
There is a refactor technique to deal with nested **if** clauses and improve the readability of the code, it's called **guard clauses**, you can read about it here https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html

Reply:
( I know that that code is not part of the final PR, so it's only **FYI**